client/blocks/reader-feed-header/index.jsx

```diff
@@ -13,25 +13,14 @@ import ReaderFollowButton from 'reader/follow-button';
 import { isAuthorNameBlacklisted } from 'reader/lib/author-name-blacklist';
 import HeaderBack from 'reader/header-back';
 import { getSiteDescription, getSiteName, getSiteUrl } from 'reader/get-helpers';
-import Gridicon from 'gridicons';
 import SiteIcon from 'blocks/site-icon';
```

Comment:
The `SiteIcon` import? I think we still need that :)

`Gridicon` has been removed though 👍